src/framing/Frame.cpp

```diff
@@ -95,78 +95,120 @@ std::ostream& operator<<(std::ostream& os, ErrorCode errorCode) {
   return os << "ErrorCode(" << static_cast<uint32_t>(errorCode) << ")";
 }
 
-std::ostream& operator<<(std::ostream& os, FrameFlags frameFlags) {
-  // TODO Match the Flag names with the AllowedFlags in the Frame declarations
+constexpr const char sFrameFlags_EMPTY[] = "0x00";
```

Comment:
These should be `const char*`.  Normally `char[]` is reserved for mutable character buffers.